quickstart.org

```diff
@@ -4,46 +4,73 @@
 #+include: "./resources/preamble.html" export html
 
 * Getting started with Org-mode
-There's a lot that Org-mode /can/ do for you, but you don't need much to get started.
+In its simplest form, Org is like any other markup language.
+You can write [[https://orgmode.org/worg/dev/org-syntax.html][Org syntax]] in any text editor.
 
-You can start writing Org syntax in any plain text editor, however to see what
-makes it /great,/ one really needs have an editor with rich support.
+However, to experience the full potential of Org you need an editor
+that transforms Org syntax into a living, interactive document.
 
 #+begin_center
 #+attr_html: :class btn
 *[[file:install.org][Install Org]]*
 #+end_center
 
-At the start of a file, it is common to set a title and author
-(these are two of the [[https://orgmode.org/manual/Export-Settings.html][export options]] --- think file properties).
+* Preamble
+At the start of a file (before the first headline), it is common to
+set the title, author, and other
+[[https://orgmode.org/manual/Export-Settings.html][export options]].
 #+BEGIN_SRC org
-,#+title: My file
-,#+author: myself
+,#+title: The glories of Org
+,#+author: A. Org Writer
 #+END_SRC
-* Headings
-Lines starting with ~*~ are headings. A single ~*~ denotes 1st-level heading, ~**~
-denotes a 2nd-level heading, etc.
+
+* Headlines
+Lines that start with and asterisk ~*~ are [[https://orgmode.org/worg/dev/org-syntax.html#Headlines_and_Sections][headlines]].
+
+A single ~*~ denotes a 1st-level headline, ~**~ denotes a 2nd-level headline, etc.
 #+BEGIN_SRC org
 ,* Welcome to Org-mode
 ,** Sub-heading
-   Every extra ~*~ brings you to a lower heading level.
+Each extra ~*~ increases the depth by one level.
+#+END_SRC
+
+In their simplest form headlines are just the start of a section.
+However, any headline can also become a [[https://orgmode.org/manual/TODO-Items.html][TODO item]].
+
+TODO items are the basic building block that Org uses to track and
+organize all kinds of tasks.
+
+To add a TODO item to a heading, start the heading with a TODO
+keyword such as ~TODO~ or ~HOLD~.
+
+#+BEGIN_SRC org
+,* TODO Promulgate Org to the world
+,** TODO Create a quickstart guide
 #+END_SRC
 
-To make a header into a task, just start the heading with a task keyword like
-~TODO~ or ~HOLD~.
-* Text styles
-With org, *bold* /italic/ and _underline_ are all nice and simple. ~code~ and =verbatim=
-are also nice and easy. If you must, +strikethrough+ is also available.
-_/*Combine*/_ these as you'd like.
+* Markup
+To mark up text in Org, simply surround it with one or more marker characters.
+*bold* /italic/ and _underline_ are fairly intuitive, and the ability to use
++strikethrough+ is a plus.  You can +_/*combine*/_+ the basic markup in any
```

Comment:
I'm not sure that striking `+_/*combine*/_+` is a good idea, though I do appreciate that it demonstrates 4 layers of markup instead of 3.

Reply:
I wondered about this as well. It seemed like it could confuse but wanted to check. Thus with check, will remove.